nixpkgs_review/github.py

```diff
@@ -1,23 +1,35 @@
+import sys
 import json
+import os
+import urllib.error
 import urllib.parse
 import urllib.request
 from collections import defaultdict
 from typing import Any, DefaultDict, Dict, Optional, Set
 
+import backoff
+
+OWNER = "NixOS"
+REPO = "nixpkgs"
+
 
 def pr_url(pr: int) -> str:
-    return f"https://github.com/NixOS/nixpkgs/pull/{pr}"
+    return f"https://github.com/{OWNER}/{REPO}/pull/{pr}"
 
 
 class GithubClient:
     def __init__(self, api_token: Optional[str]) -> None:
         self.api_token = api_token
 
+    @backoff.on_exception(backoff.expo, urllib.error.HTTPError, max_time=60)
```

Comment:
Does this mean it waits 60 seconds per request or 60 seconds in total? Might be a bit high.

Reply:
It's exponential backoff. It will increase waiting time between each try up to one minute.